### PR TITLE
 #143 FEATURE - Search MaintenanceEvents on the DashboardView

### PR DESCRIPTION
**Pull request by sergpryimachuk**
# What it Does
- #143 
- Adds ability to search `MaintenanceEvent`s on the `DashboardView` through the titles.

# How I Tested
1. Run the app.
2. Add some `MaintenanceEvent`s.
3. Tap on the search bar and type a query.

### CODE REVIEW

File: Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift [hunk 1]
```diff
@@ -20,9 +20,10 @@ struct DashboardView: View {
     }
     
     var body: some View {
+        @Bindable var viewModel = viewModel
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift]: this doesn't need to be here, it's already defined above

File: Basic-Car-Maintenance/Shared/Dashboard/ViewModels/DashboardViewModel.swift [hunk 2]
```diff
@@ -29,6 +30,14 @@ class DashboardViewModel {
         }
     }
     
+    var viewEvents: [MaintenanceEvent] {
```
Comment (mikaelacaron) [inline Basic-Car-Maintenance/Shared/Dashboard/ViewModels/DashboardViewModel.swift]: can you change this variable name, instead call it `searchedEvents`

**Review by mikaelacaron — CHANGES_REQUESTED** [review]
Great work! Just a few small changes. After you fix those, can you click the re-review button again, and I'll be notified, thanks!


Re-review button

**Comment by sergpryimachuk** [comment]
Voilà, fixed! Thanks for the review 😊

**Review by mikaelacaron — APPROVED** [review]
Great work!!

**Pull request merged**